src/ducks/reimbursements/NoReimbursedExpenses.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+
+import { GROUP_DOCTYPE } from 'doctypes'
+import Padded from 'components/Padded'
+import StoreLink from 'components/StoreLink'
+import { KonnectorChip } from 'components/KonnectorChip'
+import { getCategoryName } from 'ducks/categories/categoriesMap'
+import Caption from 'ducks/reimbursements/Caption'
+
+const NoReimbursedExpenses = ({ hasHealthBrands, doc }) => {
+  const { t } = useI18n()
+
+  const categoryName = doc.categoryId ? getCategoryName(doc.categoryId) : null
+
+  let message
+  if (doc._type === GROUP_DOCTYPE) {
+    message = t('Reimbursements.noReimbursed.group')
+  } else if (categoryName) {
+    message = t(`Reimbursements.noReimbursed.${categoryName}`)
+  } else {
+    message = t('Reimbursements.noReimbursed.generic')
+  }
+
+  return (
+    <Padded className="u-pv-0">
+      <Caption>{message}</Caption>
+      {!hasHealthBrands && categoryName === 'healthExpenses' && (
+        <StoreLink type="konnector" category="insurance">
+          <KonnectorChip konnectorType="health" />
+        </StoreLink>
+      )}
+    </Padded>
```

Comment:
we need to test the 3 messages possibilities, the categoryName filled with health Expenses, filled with 'toto' and not filled, and hadHealthBrands = false

Reply:
l'objet de la PR est de faire une refacto rapide, ce composant est aussi suffisamment simple pour qu'on s'évite de ne pas faire confiance à react ou à js et tester 3 if qui set une variable

Reply:
Peut-être que cette PR n'a pas cet objectif, mais si on bosse sur l'application aujourd'hui et qu'on refacto, c'est parce qu'on a des bugs et qu'on veut stabiliser l'app et être plus confiant. Ça passe par l'ajout de test pour éviter les régressions et pour documenter les comportements

Reply:
d'ac, c'est juste que l'estimation initiale ne prenait pas ça en compte, mais si ce n'est pas un souci...

Par contre pour ce cas précis de 3 if, un test vous semble pertinent ??? 🤔

Reply:
la philosophie de TDD veut un test par "if", donc oui.

Après, je te fais confiance pour passer efficacement du temps pour protéger le composant. Si certaines branches du if ne sont pas si importantes, ... => nit ;-)

Reply:
Ok. Pour faire efficace j'ai sorti le if dans une méthode.